framework/src/application/node/node.ts

```diff
@@ -541,9 +543,15 @@ export class Node {
 		});
 
 		this._transactionPool = new TransactionPool({
-			// FIXME: #5619 any should be removed
+			baseFees: this._options.genesisConfig.baseFees.map(fees => ({
+				...fees,
+				baseFee: BigInt(fees.baseFee),
+			})),
+			minFeePerByte: this._options.genesisConfig.minFeePerByte,
 			// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-			applyTransactions: this._processor.verifyTransactions.bind(this._processor) as any,
+			applyTransactions: this._processor.verifyTransactions.bind(
```

Comment:
i think verifyTransactions takes 2 inputs `transactions` and `stateStore`, so it should be
```
applyTransactions: async (transactions) => {
  const stateStore = this._chain.newStateStore();
  return this._processor(verifyTransactions(transactions, stateStore);
}
```